src/gurt/telemetry.c

```diff
@@ -96,7 +95,7 @@ d_tm_find_child(uint64_t *shmem_root, struct d_tm_node_t *parent, char *name)
 	if (child != NULL)
 		client_name = d_tm_conv_ptr(shmem_root, child->dtn_name);
 
-	while ((child != NULL) &&
+	while ((child != NULL) && (client_name != NULL) &&
 	       strncmp(client_name, name, D_TM_MAX_NAME_LEN) != 0) {
 		child = d_tm_conv_ptr(shmem_root, child->dtn_sibling);
 		client_name = NULL;
```

Comment:
Not a new change, but this line isn't necessary, is it?

Reply:
The file failed a coverity scan and wanted that change.